### ISSUE TITLE
German Keyboard & Chrome - Backticks destroy backspace

### ISSUE THREAD

**Issue by JanLoebel** [ianstormtaylor/slate#1761]
#### Do you want to request a _feature_ or report a _bug_?
Bug

#### What's the current behavior?
- If I use my german keyboard to write 2 backticks after each other in Chrome (v65.0.3325.181), backspace is broken afterward. Firefox instead works. I've tested it also with draftjs and there it works in the same browser.

How to reproduce:
- Select a german keyboard layout
- Open: http://slatejs.org/#/rich-text in Chrome
- Go to the last line, press twice the backtick, some text afterward
- Try to use backspace to delete the text again.

Gif of behavior:
![kapture 2018-04-06 at 8 25 54](https://user-images.githubusercontent.com/1765067/38406292-97ae0e0a-3974-11e8-86a8-15e8021b237d.gif)

OS: OSX 10.13.3, Mac Book Pro 2017

#### What's the expected behavior?
- Backspace works as before

**Comment by JanLoebel** [comment]
Any ideas for this? Is there anything I could help debugging?

**Comment by JanLoebel** [comment]
Can anybody confirm that this doesn't happen anymore in the current master branch? I've tried it several minutes ago and it was working quite well.

In other projects like: https://canner.github.io/canner-slate-editor/  I can still reproduce it.. So my guess is that it has been fixed with another issue resolved in the last several days.

If anybody could confirm this with a german keyboard/keymapping it would be great :)
